src/main/resources/templates/scala-lang/scalaClassGraphqlResponseProjection.ftl

```diff
@@ -7,6 +7,9 @@ import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseProjection
 <#if equalsAndHashCode>
 import java.util.Objects
 </#if>
+<#if fields?has_content && generateAllMethodInProjection>
+import scala.collection.mutable.HashMap
```

Comment:
Scala doesn't have `getOrDefault ` method. If you want to use it, you have to use Java HashMap.
```
import java.util.{HashMap => JHashMap}
val msp = new JHashMap()
```
or use `getOrElse` in scala.

Reply:
Good improvement, do you know how to build these locally? I am getting an error that the 5.0.1-SNAPSHOT cannot be found

Reply:
for sbt
![image](https://user-images.githubusercontent.com/29496873/114889782-fc1e4280-9e3c-11eb-8c1f-7607b16618de.png)

for gradle
![image](https://user-images.githubusercontent.com/29496873/114889840-0cceb880-9e3d-11eb-8d4b-6b13e7a96d83.png)

Reply:
I changed it to the scala native HashMap with getOrElse